client/subcomponents/events/EventForm/EventForm.jsx

```diff
@@ -40,89 +40,106 @@ export default function EventForm(props) {
 
   return (
     <>
-      <section>
-        <h2 className="form-title">{props.action}</h2>
-        <form className="form-content" onSubmit={formik.handleSubmit}>
-          <div className="field">
-            <label htmlFor="title" className="label">
-              Event Title
-            </label>
-            {formik.errors.title && formik.touched.title ? (
-              <p className="inputError">{formik.errors.title}</p>
-            ) : null}
-            <input
-              className="form-box"
-              id="title"
-              name="title"
-              type="text"
-              placeholder="event title"
-              onChange={formik.handleChange}
-              value={formik.values.title}
-            />
+      <section className="mx-auto max-w-md mt-9 px-8">
+        <div className="">
+          <h2 className="form-title text-darkBlue text-[24px] font-serif py-3">
+            {props.action}
+          </h2>
 
-            <label htmlFor="date" className="label">
-              Date
-            </label>
-            {formik.errors.date && formik.touched.date ? (
-              <p className="inputError">{formik.errors.date}</p>
-            ) : null}
-            <input
-              className="form-box"
-              id="date"
-              name="date"
-              type="date"
-              role="date"
-              placeholder="date"
-              onChange={formik.handleChange}
-              value={formik.values.date}
-            />
+          <form
+            className="form-content text-[16px] text-black"
+            onSubmit={formik.handleSubmit}
+          >
+            <div className="field">
+              <label htmlFor="title" className="label">
+                Event Title
+              </label>
+              <div>
+                {formik.errors.title && formik.touched.title ? (
+                  <p className="inputError">{formik.errors.title}</p>
+                ) : null}
+                <input
+                  className="form-box border border-[1.5px] border-blue rounded-[4px] p-2 w-full mb-5"
+                  id="title"
+                  name="title"
+                  type="text"
+                  onChange={formik.handleChange}
+                  value={formik.values.title}
+                />
+              </div>
+              <label htmlFor="date" className="label">
+                Date
+              </label>
+              <div>
```

Comment:
double check if any `<div>` tags that do not have classnames added need to be used or not. please also have a look into "semantic html" - this is great for user accessibility and code readability, see if there are any tags that could be replaced with more descriptive semantic tags in this file :) [semantic html guide](https://www.freecodecamp.org/news/semantic-html5-elements/#:~:text=Semantic%20HTML%20elements%20are%20those,content%20that%20is%20inside%20them.)